src/plugins/query_enhancements/public/types.ts

```diff
@@ -7,7 +7,7 @@ import { CoreSetup, CoreStart } from 'opensearch-dashboards/public';
 import { DataSourcePluginStart } from 'src/plugins/data_source/public';
 import { DataPublicPluginSetup, DataPublicPluginStart } from '../../data/public';
 import { UsageCollectionSetup } from '../../usage_collection/public';
-
+import { AssistantPublicPluginStart } from '../../../../plugins/dashboards-assistant/public';
```

Comment:
actually think make break it since this is an optional plugin it but is trying to access it the file directly. in the case where there is no dashboards assist plugin installed what happens @Qxisylolo ? does the application crash?

Reply:
I think this will break the build if the plugin was not pulled in to the `plugins` folder, OSD should not depends on 3rd party plugins.

Reply:
ok, I see, I will call the API endpoint , Thanks for all the comments